.babelrc.json

```
@@ -0,0 +1,17 @@
+// Storybook でのみ使用
+{
+  "sourceType": "unambiguous",
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "chrome": 100
+        }
+      }
+    ],
+    "@babel/preset-typescript",
+    "@babel/preset-react"
+  ],
```

標準でモダンブラウザサポートしてそうなので、消してよさそうでした!（消しても動作的には問題なさそうに見えました!
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#modern-browser-support

```suggestion
  "presets": ["@babel/preset-env", "@babel/preset-typescript", "@babel/preset-react"],
```